modules/crud/sql/create.go

```diff
@@ -16,15 +16,15 @@ import (
 
 // Create inserts a document (or multiple when op is "all") into the database
 func (s *SQL) Create(ctx context.Context, project, col string, req *model.CreateRequest) error {
-	sqlQuery, args, err := s.generateCreateQuery(ctx, project, col, req)
+	sqlQuery, args, err := s.GenerateCreateQuery(ctx, project, col, req)
```

Comment:
Don't export this function

Reply:
I tried to not export these func
But while running go test command
It says not test files found or undefined function name

Reply:
so the name of the testing function in *_test.go will always start with "Test". So the test functions will always be exported but the function to be tested can be un-exported.